projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts

```diff
@@ -71,27 +71,39 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
     this.tableDataSource = new TableLocalDataSource(this.tableData$, this.getDatasourceConfig());
     this.keywordsService = this.executionContext.keywordsContext;
     const keywordsContext = this.executionContext.keywordsContext;
```

Comment:
Why are we saving the context in a class varaible as well as in a local const?

Reply:
for no reason